code/modules/mob/living/simple_animal/hostile/wendigo.dm

```diff
@@ -0,0 +1,224 @@
+
+
+/mob/living/simple_animal/hostile/wendigo
+	name = "wendigo"
+	desc = "Thought to be just a cautionary tale, now moreso than ever."
+	icon_state = "wendigo"
+	icon_living = "wendigo"
+	icon_dead = "wendigo_dead"
+
+	health = 150
+	maxHealth = 150
+	speed = 1
+	stat_attack = DEAD //Gotta chow down somehow
+	vision_range = 12 //Slightly larger vision range
+	harm_intent_damage = 8
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	meat_amount = 1 //Only one piece of meat
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/wendigo
+	var/list/names = list() //List of names of the people it's eaten
+	var/consumes = 0 //How many people it has eaten
+	environment_smash = 1
+	speak = list("Help!","Help me!","Somebody help!","Get over here, quickly!")
+
+
+/mob/living/simple_animal/hostile/wendigo/CanAttack(var/atom/the_target)
+	var/list/target_prox = view(the_target, vision_range)
+	for(var/obj/machinery/space_heater/campfire/fire in target_prox)
+		var/dist = get_dist(the_target, fire)
+		if(dist < (fire.light_range*2))//Just sitting on the edge of the fire
+			return
+
+	return ..(the_target)
+
+/mob/living/simple_animal/hostile/wendigo/AttackingTarget()
+	if(!target)
+		return
+
+	if(ismob(target))
+		var/mob/living/mob_target = target
+		if(mob_target.isDead() && !istype(mob_target, /mob/dead/observer))
+			visible_message("<span class = 'notice'>\The [src] starts to take a bite out of \the [target].</span>")
+			stop_automated_movement = 1
+			if(do_after(src, mob_target, 50, needhand = FALSE))
+				playsound(get_turf(src), 'sound/weapons/bite.ogg', 50, 1)
+				var/damage = rand(melee_damage_lower, melee_damage_upper)
+				mob_target.adjustBruteLoss(damage)
+				if(health < maxHealth)
+					health = min(maxHealth,(health+damage))
+
+				if(ishuman(mob_target))
+					if(mob_target.health < -400)
+						visible_message("<span class = 'warning'>\The [src] is trying to eat \The [mob_target]!</span>","<span class = 'warning'>You hear crunching.</span>")
+						if(do_after(src, mob_target, 50, needhand = FALSE))
+							consumes += 1
+							names += mob_target.real_name
+							qdel(mob_target)
+
+			return
+	. =..()
+
+/mob/living/simple_animal/hostile/wendigo/Life()
+	/*Check evolve like zombies, run away from fire,
+	mimic speech if anyone's nearby but just out of eyesight
+	*/
+	..()
+	if(!isDead())
+		check_evolve()
+
+		var/list/can_see = view(src, vision_range)
+
+		for(var/obj/machinery/space_heater/campfire/fire in can_see)
+			var/dist = get_dist(src, fire)
+			if(dist < (fire.light_range*2))
+				walk_away(src,fire,(fire.light_range*2),move_to_delay)
+
+
+		if(names.len)
+			speak_chance = 15
+
+
+/mob/living/simple_animal/hostile/wendigo/GetVoice()
+	if(names.len)
+		return pick(names)
+	else
+		speak_chance = 0
+		return name
+/mob/living/simple_animal/hostile/wendigo/proc/check_evolve()
+	return
+
+/mob/living/simple_animal/hostile/wendigo/human
```

Comment:
Rather than it just being another naked human, use the wearing code from mannequins.

Could use that for the turned zombies, come to think of it